app/scripts/lib/setupWeb3.js

```diff
@@ -44,21 +44,26 @@ export default function setupWeb3 (log) {
       }
 
       if (shouldLogUsage) {
+        const name = typeof key === 'string'
+          ? key
+          : `typeof ${typeof key}`
```

Comment:
We should probably DRY this up, but I'm not gonna block on it.

Reply:
Updated!